src/components/order-summary/_order-summary.scss

```diff
@@ -0,0 +1,147 @@
+.bx--order-summary {
+  @include font-size('14');
+  background-color: $ui-01;
+  padding-bottom: 1.25rem;
+  width: 18rem;
+  height: 100%;
+  min-height: 100vh;
```

Comment:
Yeah I'm a little worried if we set it to 100vh - softlayer has both a header _and_ a footer, it might just be better to say leave that height `100%` and ask the consumer to set the appropriate height on the container element